simulationdataschema/model_system.py

```diff
@@ -325,11 +322,11 @@ class AtomicCell(GeometricSpace):
         shape=["*"],
         # TODO improve description
         description="""
-        Wyckoff letters associated with each atom position.
+        Wyckoff letters associated with each atom.
```

Comment:
My comment about the symmetry endures.

Reply:
I added the comment saying that we have to clarify whether this belongs to `Symmetry`.